instructure/canvas-ios
# Change API for all quizzes.

refs: MBL-14898
affects: Student, Teacher
release note: Add New Quizzes type assignments to Quiz Index screen.

test plan:
- Have a published course with a New Quizzes assignment.
- Login as the student or teacher on the iOS app.
- Navigate to the quizzes link in the course navigation.
- New Quizzes assignment should appear on the quiz index screen.
